patches/minecraft/net/minecraft/entity/Entity.java.patch

```diff
@@ -262,7 +261,23 @@
           }
        } else {
           return null;
-@@ -2320,9 +2357,10 @@
+@@ -2124,7 +2161,15 @@
+    }
+ 
+    @Nullable
++   @Deprecated // Forge: Use ITeleporter safe version
+    protected PortalInfo func_241829_a(ServerWorld p_241829_1_) {
++      return this.getPortalInfo(p_241829_1_, p_241829_1_.func_85176_s());
++   }
++
++   @Nullable
++   protected PortalInfo getPortalInfo(ServerWorld p_241829_1_, net.minecraftforge.common.util.ITeleporter teleporter) {
```

Comment:
Going to opt to not mark it as nullable since it still technically shouldn't be null and it isn't really "handled" properly if it is.

Reply:
It might be worth removing the != null check then to make it a bit more readable and obvious that it follows the parameters are nonnull by default and that it wasn't just a missed nullable annotation

Reply:
Yeah that's a valid point. I wasn't sure if I should leave it there or not.